### PR TITLE
Fingerprint for Lexus NX Hybrid 2019

### PR DESCRIPTION
**Pull request by neneboe**
Added fingerprint for Lexus NX Hybrid 2019. All it needed was the engine fwVersion.

My dongle id is ed54b42db6f29263.

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ class CAR:
   LEXUS_IS = "LEXUS IS 2018"
   LEXUS_CTH = "LEXUS CT HYBRID 2018"
   RAV4H_TSS2 = "TOYOTA RAV4 HYBRID 2019"
-  LEXUS_NXH = "LEXUS NX HYBRID 2018"
+  LEXUS_NXH = "LEXUS NX HYBRID 2018 & 2019"
```
Comment (VirtuallyChris) [inline selfdrive/car/toyota/values.py:56]: Revert this change

**Review by VirtuallyChris — CHANGES_REQUESTED** [review]

File: selfdrive/car/toyota/values.py (hunk 1 above)
Comment (VirtuallyChris) [reply selfdrive/car/toyota/values.py:56]: Just in the readme, the platform is the 2018 and up but it identifies as 2018. That's why all cars only have 1 year listed

Comment (neneboe) [reply selfdrive/car/toyota/values.py]: Understood. Thanks, Chris.
